projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -107,15 +107,29 @@ public function prepare_menu_for_response( array $menu ) {
 		 * See: https://github.com/Automattic/wp-calypso/blob/ebde236ec9b21ea9621c0b0523bd5ea185523731/client/state/admin-menu/schema.js
 		 */
 		foreach ( $menu as $menu_item ) {
-			$item = $this->prepare_menu_item( $menu_item );
+
+			/**
+			 * Logic extracted from _wp_menu_output() core function
+			 * Based $admin_is_parent we determine if the submenu items should be pointing to the admin.php
```

Comment:
There are [WordPress-specific phpcs standards](https://github.com/WordPress/WordPress-Coding-Standards) that include documentation standards, which might be helpful her?